plugins/api-docs/src/components/ApisCards/ConsumedApisCard.tsx

```diff
@@ -85,8 +98,16 @@ export const ConsumedApisCard = (props: {
             APIs.
           </Typography>
           <Typography variant="body2">
-            <Link to="https://backstage.io/docs/features/software-catalog/descriptor-format#specconsumesapis-optional">
-              Learn how to change this.
+            <Link
+              to="https://backstage.io/docs/features/software-catalog/descriptor-format#specconsumesapis-optional"
+              externalLinkIcon={
```

Comment:
I wonder if we should just move this icon entirely to the link component and always use `app.getSystemIcon('externalLink') || OpenInNew` there instead, rather than this implementation leaking everywhere?

Feels nicer the fix this at a DLS level rather than just opening up the API? :pray:

Reply:
We can do this, I was thinking that you might not always want to use the icon with the link for some reason and therefore decided to open up the api so that it can be optional. 

Can make it so that the prop is still there but that we make it a boolean so if the prop is passed through we can render the icon, sounds good?